app/assets/javascripts/comments.js

```diff
@@ -85,11 +86,16 @@ $(() => {
         </div>
         <div class="actions">
           <input type="submit" class="button is-outlined" value="Post" />
+          <input type="button" name="js-discard-edit" id="js-discard-edit-${commentId}" value="Discard Edit" class="button is-danger is-outlined" />
```

Comment:
Can we make this `class="js-discard-edit" data-comment-id="${commentId}"` instead of packing it all into the ID attribute?